helm_deploy/use-of-force/values.yaml

```diff
@@ -12,4 +12,3 @@ generic-data-analytics-extractor:
 
 ingress:
   v1_2_enabled: true
-  v0_47_enabled: true
```

Comment:
Deleting this flag and deleted the ingress.yaml file. I thought I'd keep the new one named as v1_2 as it keeps it in par with the generic service. Hopefully makes it easier when / if we migrate to it.